ee_tests/README.md

```diff
@@ -5,30 +5,30 @@
 The goal of this repository is to provide automated end-to-end (E2E) tests for production
 and stage environment of OpenShift.io.
 
+To see the latest tests results in the CI visit our Wiki page: https://github.com/fabric8io/fabric8-test/wiki
+
 The default test suite is called `smoketest` which serves as a monitoring of the product.
 It is expected to run relatively fast therefore this suite will *NOT* test multiple boosters,
 multiple pipeline types, advanced interactions with Che etc., this should be done
 by unit, functional or integration tests of particular components.
 
 ### Smoketest Suite ###
 
-The `smoketest` suite is being executed by CentOS CI on prod-preview, on every prod cluster and for both
-`beta` and `released` feature levels.
-
-* [Jenkins jobs on prod-preview](https://ci.centos.org/view/Devtools/search/?q=devtools-test-e2e-prod-preview.openshift.io-smoketest)
-* [Jenkins jobs on production clusters](https://ci.centos.org/view/Devtools/search/?q=devtools-test-e2e-openshift.io-smoketest)
-
-The `smoketest` suite tests (or will be testing) *basic* functionality of *all* OpenShift.io components, i.e.
+The `smoketest` suite tests *basic* functionality of *all* OpenShift.io components
 
 1. logs into the platform
 2. creates a new space
-3. creates a new Vert.x project
+3. creates a new project
 4. opens Che workspace
 5. verifies that pipeline behaves correctly
 6. verifies deployed application in stage and run environments
-7. verifies data in deployments page and analytics report
+7. verifies data on the dashboard page and analytics report
```

Comment:
Do we want to mention that it also creates a work item in beta config?

Reply:
OK